From c23b6d11057b947ffba671afa24309c22e5deb15 Mon Sep 17 00:00:00 2001
From: Sean Busbey <busbey@apache.org>
Date: Sun, 22 Feb 2015 00:47:33 -0600
Subject: [PATCH] HBASE-13045 Addendum for Hadoop 1.1 compat.

Use Text instead of NullWritable to meet TableMap type requirements.
---
 .../org/apache/hadoop/hbase/mapred/TestTableInputFormat.java     |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hbase-server/src/test/java/org/apache/hadoop/hbase/mapred/TestTableInputFormat.java b/hbase-server/src/test/java/org/apache/hadoop/hbase/mapred/TestTableInputFormat.java
index ddefce6ece122c2b779794be19a85ddaf70b2ec1..a79e499dc47af0936a0aa21688b379d3c98d3faa 100644
--- a/hbase-server/src/test/java/org/apache/hadoop/hbase/mapred/TestTableInputFormat.java
+++ b/hbase-server/src/test/java/org/apache/hadoop/hbase/mapred/TestTableInputFormat.java
@@ -49,7 +49,7 @@ import org.apache.hadoop.hbase.mapreduce.MapreduceTestingShim;
 import org.apache.hadoop.hbase.io.ImmutableBytesWritable;
 import org.apache.hadoop.hbase.testclassification.LargeTests;
 import org.apache.hadoop.hbase.util.Bytes;
-import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.io.Text;
 import org.apache.hadoop.mapred.JobClient;
 import org.apache.hadoop.mapred.JobConf;
 import org.apache.hadoop.mapred.JobConfigurable;
@@ -342,7 +342,7 @@ public class TestTableInputFormat {
         .findCounter(TestTableInputFormat.class.getName() + ":value", "value bbb").getCounter());
   }
 
-  public static class ExampleVerifier implements TableMap<NullWritable, NullWritable> {
+  public static class ExampleVerifier implements TableMap<Text, Text> {
 
     @Override
     public void configure(JobConf conf) {
@@ -350,7 +350,7 @@ public class TestTableInputFormat {
 
     @Override
     public void map(ImmutableBytesWritable key, Result value,
-        OutputCollector<NullWritable,NullWritable> output,
+        OutputCollector<Text,Text> output,
         Reporter reporter) throws IOException {
       for (Cell cell : value.listCells()) {
         reporter.getCounter(TestTableInputFormat.class.getName() + ":row",
-- 
1.7.10.2 (Apple Git-33)

